Failed on delivery (Build a docker image) "chown: invalid user: 'node'"

GitHubにpushしたときのGitHub Actionsでのdelivery -> Build a docker image がエラーが発生して失敗するようになりました。

https://github.com/vivliostyle/vivliostyle-pub/actions/runs/2467203256

次のエラーが発生してます：

```
chown: invalid user: 'node'
The command '/bin/sh -c chown -R node tmp' returned a non-zero code: 1
Error: Process completed with exit code 1.
```


Dockerfile
https://github.com/vivliostyle/vivliostyle-pub/blob/master/cloud-run/Dockerfile
の次のところ、

https://github.com/vivliostyle/vivliostyle-pub/blob/bfdeb318c3bb02ace2c98527a9a615e96d35b9dc/cloud-run/Dockerfile#L15-L16

ここで "chown: invalid user: 'node'" のエラーになってます。`USER node` はそのあとにあります:

https://github.com/vivliostyle/vivliostyle-pub/blob/bfdeb318c3bb02ace2c98527a9a615e96d35b9dc/cloud-run/Dockerfile#L25

--- a/cloud-run/Dockerfile
+++ b/cloud-run/Dockerfile
@@ -1,8 +1,6 @@
 FROM ghcr.io/vivliostyle/cli:latest
 LABEL maintainer "Vivliostyle Foundation <mail@vivliostyle.org>"
 
-USER root
-
 RUN set -x \
   && apt-get update \
   && apt-get install -y git
@@ -13,7 +11,6 @@ RUN npm install
 COPY . .
 
 RUN mkdir tmp; exit 0
-RUN chown -R node tmp
 
 ARG GH_APPS_ID
 ARG GH_APPS_PRIVATEKEY
@@ -22,6 +19,5 @@ ENV GH_APPS_PRIVATEKEY=$GH_APPS_PRIVATEKEY
 
 RUN npm run build
 
-USER node
 ENTRYPOINT [ "/bin/sh", "-c" ]
 CMD [ "npm start" ]

